Firmware/stepper.cpp

```diff
@@ -393,18 +389,20 @@ FORCE_INLINE void stepper_next_block()
     }
 
     if (current_block->flag & BLOCK_FLAG_DDA_LOWRES) {
-      counter_x.lo = -(current_block->step_event_count.lo >> 1);
-      counter_y.lo = counter_x.lo;
-      counter_z.lo = counter_x.lo;
-      counter_e.lo = counter_x.lo;
+      const int16_t value = -(current_block->step_event_count.lo >> 1);
+      for (uint8_t axis = 0; axis < NUM_AXIS; axis++)
```

Comment:
you can even iterate from the back which saves 2-4 bytes more
```
for (uint8_t axis = NUM_AXIS-1; axis >=0 ; --axis){
    counter[axis].lo = value;
}
```
Caches and memory ordering is not an issue on the AVR IMHO

Reply:
@petrubecheru Thanks but compiling with `PF-build.sh` the usage of Flash and RAM are the same before and after.